packages/vm/src/imports.rs

```diff
@@ -75,6 +89,15 @@ pub fn native_humanize_address<A: Api, S: Storage, Q: Querier>(
     do_humanize_address(&env, source_ptr, destination_ptr)
 }
 
+pub fn native_verify_secp256k1<A: Api, S: Storage, Q: Querier>(
+    env: &Environment<A, S, Q>,
+    hash_ptr: u32,
+    signature_ptr: u32,
+    pubkey_ptr: u32,
+) -> VmResult<u32> {
+    do_verify_secp256k1(&env, hash_ptr, signature_ptr, pubkey_ptr)
```

Comment:
```suggestion
    do_verify_secp256k1(env, hash_ptr, signature_ptr, pubkey_ptr)
```

This is a `&&Environment<A, S, Q>` in the other places as well (which we don't need/want).

Reply:
👍🏼 Good eye.

Reply:
This is not what is being done in the other fns, though (just so you know).

Reply:
Right, it is wrong in the other places as well. This happened as part of massive refactorings during the Wasmer 1.0 upgrade. But it seems to work still. No idea why && is often accepted when you just need a &.

Reply:
I think that it's not that `&&` is accepted, but most probably, that adding an `&` in front of a reference, for a parameter that already expects a reference, is optional; i.e. it amounts to syntaxtic sugar / for clarity stuff.